### PR TITLE
NF: Add Nullability Annotations to Scheduler

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Adding some @NonNull and @Nullable in the scheduler

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java [hunk 1]
```diff
@@ -99,27 +99,24 @@
     protected int mRevCount;
 
     private int mNewCardModulus;
-    /** Next time the queue is reset, takes into account that this card is in the reviewer and so should not be added
-     * to queue. I.e. that is mCurrentCard but should not be discarded by reset.
-     * */
-    protected Card mCardNotToFetch = null;
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java:105]: Why was this removed?

**Review by mikehardy — APPROVED** [review]
LGTM assuming it compiles + passes the lint where the annotations are exercised

**Review by david-allison-1 — APPROVED** [review]
One question, then looks good to me

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java:105]: As indicated in the commit which removes it, this variable is not used anymore. It's a remaining for the complex version of preLoadNextCard

**Comment by Arthur-Milchior** [comment]
> passes the lint where the annotations are exercised

Where can I check it ?

**Comment by mikehardy** [comment]
that's `./gradlew :AnkiDroid:lintRelease` - but Travis runs it, it's the 4th runner in the first block on travis
